src/vivarium_csu_alzheimers/data/loader.py

```diff
@@ -372,6 +372,21 @@ def transform_group_index_J_BBBM(DUR, W, N, index_p0):
     )
 
 
+def loc_any_group(df, index, val):
+    # get row of dataframe (population, incidence or mortality) by index tuple.
+    # if age group does not exist, set row to val
+
+    age = index[-4]  # population df starts with location; incidence and mortality dfs don't
```

Comment:
I think there is a much cleaner way to right this function. You will need to figure out the exact syntax but I think it is easier to understand and less error prone if you do something like: 

`if age_start not in df:
    df[:] = value
else:
    old_mask = df[df.index.get_level_values('age_start') >= 95]
    df[old_mask, :] = 0.0
`